### PR TITLE
Update 06.adfs.md

### PR DESCRIPTION
**Pull request by brunoebernardi**
Suggesting the insertion of additional troubleshooting steps based on support cases submitted by customers.

### CODE REVIEW

File: docs/08.integration/06.adfs/06.adfs.md [hunk 1]
```diff
@@ -107,3 +107,35 @@ Please see the [Users and Roles](/configuration/users#mapping-groups-to-roles-an
 1. ADFS SamlResponseSignature needs to be either MessageOnly or MessageAndAssertion.  Use Get-AdfsRelyingPartyTrust command to verify or update it.
 
 ![adfsTroubleshooting](nv_adfs2.png)
+
+
+2. Time synchronization between Kubernetes Nodes x ADFS Server
+
+For authentication to be successful, the time between the K8s Nodes and the ADFS server would need to be the same.This is very likely due to time sync or clock drift issues. 
+
+The recommendation is to use an NTP server, with equal time settings between all the servers involved. Please check and confirm that both ends are synchronized and that there are no delays of more than 10 seconds. You can use linux and windows commands to check dates, times and NTP server activity.
```
Comment (nunix) [inline docs/08.integration/06.adfs/06.adfs.md:116]:
```suggestion
It's recommended to use an [NTP server](https://en.wikipedia.org/wiki/Network_Time_Protocol), with equal time settings across all servers. 

Please check and confirm that both ADFS and NeuVector hosts are synchronized and the potential delays do not exceed more than 10 seconds. You can use Linux and Windows commands to check dates, times and NTP server activity.
```

File: docs/08.integration/06.adfs/06.adfs.md [hunk 2]
```diff
@@ -107,3 +107,35 @@ Please see the [Users and Roles](/configuration/users#mapping-groups-to-roles-an
...
+
+To confirm this assumption, you can try the following WorkAround:
```
Comment (nunix) [inline docs/08.integration/06.adfs/06.adfs.md:118]:
```suggestion
:::tip
You can reload the auth times by disabling and enabling again the config in the NeuVector UI as follows:

- Log in to NeuVector with Admin User 
- Go to Settings 
- Click on the button to disable and enable the SAML setting
  - **Make sure to keep the configuration settings!**

Once the setting has been re-enabled, you can try to log in with an ADFS user. If it works, this confirms the issue was due to a time synchronization error between Kubernetes nodes and the ADFS Server. 
:::
```

Comment (nunix) [reply docs/08.integration/06.adfs/06.adfs.md:118]: I'm using a Docusaurus admonition to make the workaround stand out more

File: docs/08.integration/06.adfs/06.adfs.md [hunk 3]
```diff
@@ -107,3 +107,35 @@ Please see the [Users and Roles](/configuration/users#mapping-groups-to-roles-an
...
+
+Log in to NeuVector with Admin User -> Settings -> Click on the button to disable and enable the SAML setting (Make sure to keep the configurations). After this, try to log in with an ADFS user again. If it works, it is due to a time synchronization error between Kubernetes Nodes and the ADFS Server, because the auth times are reloaded when you disable and enable this config in the NeuVector UI. 
+
```
Comment (nunix) [inline docs/08.integration/06.adfs/06.adfs.md:121]:
```suggestion
```

File: docs/08.integration/06.adfs/06.adfs.md [hunk 4]
```diff
@@ -107,3 +107,35 @@ Please see the [Users and Roles](/configuration/users#mapping-groups-to-roles-an
...
+
+3. SAML characters must be case sensitive in NeuVector UI
+
+Attribute names are case sensitive. Make sure any SAML attribute name configured here is an exact match to the application configuration. SAML must point to the correct URL to authenticate. 
+
+All characters for all fields at NeuVector UI -> Settings -> SAML Settings must be case-sensitive.
```
Comment (nunix) [inline docs/08.integration/06.adfs/06.adfs.md:127]:
```suggestion
All the fields in `NeuVector UI -> Settings -> SAML Settings` are case-sensitive.
```

File: docs/08.integration/06.adfs/06.adfs.md [hunk 5]
```diff
@@ -107,3 +107,35 @@ Please see the [Users and Roles](/configuration/users#mapping-groups-to-roles-an
...
+
+The NeuVector controller logs contain the relevant information about authentication with the ADFS server and errors that will help identify the root cause. We recommended recreate the failed login condition and check the logs.
+
+
```
Comment (nunix) [inline docs/08.integration/06.adfs/06.adfs.md:131]:
```suggestion
```

File: docs/08.integration/06.adfs/06.adfs.md [hunk 6]
```diff
@@ -107,3 +107,35 @@ Please see the [Users and Roles](/configuration/users#mapping-groups-to-roles-an
...
+4. Make sure to enter the correct groups, certificates and protocols
+
+For successful configuration, the Identify Provider Single Sign-On URL should use the HTTPS protocol and the Identity Provider Issuer should use the HTTP protocol. ADFS SamlResponseSignature needs to be either MessageOnly or MessageAndAssertion. You will need to validate this configuration on your ADFS server and then in the NeuVector UI.
+
+The selected certificate would need to be valid and correctly generated, including CA Root and intermediate certificates. You can generate them using your trusted certificate authority, using Windows or automation tools such as LetsEncrypt.
+
+If any of these parameters are incorrect, you will receive an "Authentication Failed" error when you try to log in to NeuVector with an ADFS user using SAML authentication.
+
+
+
```
Comment (nunix) [inline docs/08.integration/06.adfs/06.adfs.md:141]:
```suggestion
The SAML settings need to match the following configuration:

| Setting | Value |
| ------- | ----- |
| Identify Provider Single Sign-On URL | Requires HTTPS protocol |
| Identity Provider Issuer | Requires HTTP protocol |
| ADFS SamlResponseSignature | Needs to be either MessageOnly or MessageAndAssertion |

:::warning attention
These settings need to be validated on your ADFS server and in the NeuVector UI.
:::

The selected certificate needs to be valid and correctly generated, including its `CA Root` and `Intermediate Certificates`. You can generate them using your trusted certificate authority, Windows or an automation tool such as [LetsEncrypt](https://letsencrypt.org/).

If any of these parameters are incorrect, you will receive an `Authentication Failed` error when you try to log in to NeuVector with an ADFS user using SAML authentication.

```

Comment (nunix) [reply docs/08.integration/06.adfs/06.adfs.md:141]: Added a table for the settings and a warning

**Review by nunix — COMMENTED** [review]
Adapted the content and rephrased some sentences.

**Comment by nunix** [comment]
@brunoebernardi thanks a lot for the content, I added my comments with some "Docusaurus styling".

Can I also ask you to copy the final content to `/versioned_docs/version-5.3/08.integratoin/06.adfs/06.adfs.md` so this is available also in the current version.

Please let me know if you have any questions.

**Comment by brunoebernardi** [comment]
Hi @nunix. Thanks a lot for your contribution and suggestions for the documentation. I've committed to your suggestions and the troubleshooting steps are more clear and understandable.

**Pull request merged**